core/src/com/unciv/ui/worldscreen/WorldScreen.kt

```diff
@@ -502,5 +504,13 @@ class WorldScreen(val viewingCiv:CivilizationInfo) : CameraStageBaseScreen() {
             displayTutorial(Tutorial.Embarking)
     }
 
+    private fun exitGamePrompt() {
+        var promptWindow = PopupTable(this)
+        promptWindow.add("Do you want to exit the game?".tr())
```

Comment:
For uniformity with the rest of the code you should probably call `addGoodSizedLabel()`

Reply:
Thank you, will correct that.